cli/tests/cli-runtime.rs

```diff
@@ -24,64 +18,4 @@ mod unix {
 
         Ok(())
     }
-
-    // async fn create_mock_proxy(
-    //     response: ResponseTemplate,
-    // ) -> Result<MockServer, Box<dyn Error + 'static>> {
-    //     let proxy = MockServer::start().await;
-    //     let platform: Option<&str> = match OS {
-    //         "linux" | "macos" | "windows" => Some(OS),
-    //         _ => None,
-    //     };
-
-    //     Mock::given(method("HEAD"))
-    //         .and(PathExactMatcher::new(format!("cli/{}", platform.unwrap())))
-    //         .respond_with(response)
-    //         .expect(1..)
-    //         .mount(&proxy)
-    //         .await;
-
-    //     Ok(proxy)
-    // }
-
-    // this test only verifies that the request is sent to the CDN as the printing
-    // is flaky given its background thread status
-    // #[async_std::test]
-    // async fn prints_if_update_is_found() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I would still like to find a way to test this if possible?

Reply:
I'll give it a shot! (after I merge this PR though)

generally speaking, I think it might be more successful in the long run, in cases like this, where instead of commenting out (no more compilation), use `[#ignore]` on the test and a comment explaining why it's being ignored, ideally with a TODO and a link to an issue?